uPortal-webapp/src/main/webapp/WEB-INF/web.xml

```diff
@@ -90,15 +90,15 @@
         <filter-class>org.jasig.cas.client.validation.Cas20ProxyReceivingTicketValidationFilter</filter-class>
         <init-param>
           <param-name>casServerUrlPrefix</param-name>
-          <param-value>${environment.build.cas.protocol}://${environment.build.cas.server}${environment.build.cas.context}</param-value>
+          <param-value>http://localhost:8080/cas</param-value>
```

Comment:
I don't believe it's a Gradle issue really -- Gradle can do filtering at least as well as Maven.
It's really about how non-portable builds are a blood-curdlingly serious issue for the long-term health of the project & community.  We really must get away from building uPortal specifically for DEV, TEST, PROD, _etc_.

So we "cut the cord" with filtering when we moved to Gradle (because it seemed a sensible opportunity).  Technically every previous portal runtime feature is still supported because you can "overlay" any file in uPortal to do exactly what you want.  But (of course), we want to discourage that too.

We will refactor every ounce of commonly-customized configuration such that it can be controlled externally.  This change is a work-in-progress.  That's okay -- we're only cutting milestone builds of uP5 at present.  We will "complete" that transformation before we have a GA release.

I changed `web.xml` in this PR becuase -- as it stands -- it's broken 100% of the time.  The change I made (so far) only fixes it in a standard, local dev deployment.

Both of @apetro's suggestions have attractiveness, though I lean toward the 2nd.  Perhaps I can get to that shortly and include it here.